test/conformance/testdata/conformance.txt

```diff
@@ -214,8 +214,13 @@ test/e2e/scheduling/predicates.go: "validates that NodeSelector is respected if
 test/e2e/scheduling/predicates.go: "validates that NodeSelector is respected if matching"
 test/e2e/storage/empty_dir_wrapper.go: "should not conflict"
 test/e2e/storage/empty_dir_wrapper.go: "should not cause race condition when used for configmaps"
+test/e2e/storage/mounted_volume_resize.go: "should verify mounted devices can be resized"
 test/e2e/storage/subpath.go: "should support subpaths with secret pod"
 test/e2e/storage/subpath.go: "should support subpaths with configmap pod"
 test/e2e/storage/subpath.go: "should support subpaths with configmap pod with mountPath of existing file"
 test/e2e/storage/subpath.go: "should support subpaths with downward pod"
 test/e2e/storage/subpath.go: "should support subpaths with projected pod"
+test/e2e/storage/volume_expand.go: "should not allow expansion of pvcs without AllowVolumeExpansion property"
```

Comment:
@PatrickLang @bclau - will this work on windows?

Reply:
They are not being run in the first place, there's a:

    framework.SkipUnlessProviderIs("aws", "gce")

in the tests BeforeEach. AFAIK, we can't promote those tests because of this, we can't allow Skips for Conformance tests.

Reply:
I'll remove this from this PR.

Reply:
All of the tests that are targetted by this PR have that Skip.